plugins/invisible-typing/src/Icon.tsx

```diff
@@ -17,43 +17,41 @@ export const Icon = (props: { type?: { analyticsName: string } }) => {
   const channelWise = cfg.get("button", true) ? cfg.get("channelWise", true) : false;
   const channelsList = cfg.get("channels", { [channelId]: globalInvisible });
   const [enabled, setEnabled] = React.useState(
-    channelWise ? channelsList[channelId] : globalInvisible,
+    channelWise ? channelsList[channelId] ?? globalInvisible : globalInvisible,
   );
   return (
-    <>
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **question (performance):** REMOVAL OF FRAGMENT IN FAVOR OF DIV WITH KEY

The change from a React fragment to a div with a dynamic key based on the 'enabled' state is an interesting choice. It would be beneficial to understand the specific reason behind this change, as it introduces a potential re-rendering trigger that wasn't present before.

```suggestion
<>
```